src/MCPServer/lib/unit.py

```diff
@@ -34,7 +78,30 @@ def setMagicLink(self,link, exitStatus=""):
         return
 
     def setVariable(self, variable, variableValue, microServiceChainLink):
-        return
+        if not variableValue:
+            variableValue = ""
+        vars = UnitVariable.objects.filter(unittype=self.unitType,
+                                           unituuid=self.UUID,
```

Comment:
Continuing with 'Why has Python taken all the good variable names?', ```vars``` is a builtin.